etc/profile-a-l/Beyond-All-Reason.profile

```diff
@@ -0,0 +1,42 @@
+# Firejail profile for Beyond All Reason
+# Description: Open source real time strategy game
+# This file is overwritten after every install/update
+# Persistent local customizations
+include beyondallreason.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/Beyond All Reason
+noblacklist ${HOME}/Downloads
+
+include disable-common.inc
+include disable-devel.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/Beyond All Reason
+whitelist ${HOME}/Beyond All Reason
+whitelist ${HOME}/Downloads
+
+caps.drop all
+ipc-namespace
+netfilter
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+#seccomp # crashes the launcher
```

Comment:
Which syscall is block?

Do you have an AMP GPU?

If we get `seccomp` to work: `seccomp.block-secondary`.

Reply:
I'm not sure. Is there an easy way to find out? Strace?

Reply:
AMP GPU? Do you mean AMD GPU? This is NVIDIA.

Reply:
> strace?

Your syslog `journalctl --grep=SECCOMP --follow`

https://github.com/netblue30/firejail/blob/3381c614cb2811b79ff7dc6d0cabfc49e27c8ae9/etc/templates/syscalls.txt#L89-L112